app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesController.scala

```diff
@@ -74,7 +76,23 @@ class ExemptionAndLossesController @Inject() (
     ) => Future[Result]
   ): Future[Result] =
     request.sessionData.flatMap(s => s.journeyStatus.map(s -> _)) match {
-      case Some((s, r @ FillingOutReturn(_, _, _, d, _))) =>
+      case Some((s, r @ FillingOutReturn(_, _, _, d, _))) if r.isFurtherReturn.contains(true) =>
```

Comment:
if you are going to call `isFurtherReturn` without the parenthesis then ideally you should remove the parenthesis from the actual function definition, so:

```scala
    def isFurtherReturn: Option[Boolean] 
```

instead of

```scala
    def isFurtherReturn(): Option[Boolean] 
```

Reply:
I don't like to use parenthesis, but if someone wants use it? I feel its good to define function like this  `isFurtherReturn()`
Then its up-to user whether use of not. If we miss it in definition, then we don't have option to use parenthesis in its usage.

Reply:
the convention is if a method has side-effects include the parenthesis else if the method is not side-effecting then do not include the parenthesis. `isFurtherReturn` is not side-effecting so by convention it should have no parenthesis.

Reply:
see this as well @rambabu-posa https://docs.scala-lang.org/glossary/index.html#uniform-access-principle

Reply:
Okay sure, thanks will pix it